data/coalition/heliarch intro.txt

```diff
@@ -1495,7 +1495,7 @@ mission "Heliarch Containment 4-A"
 			label license
 			`	He stares at you for a few seconds, almost like a statue. "Afraid I am that orders, these are. Received them directly from the Ring of Friendship, from Consul Iekie, I have. A great asset for this task, you would be, Captain, so insist upon your involvement, I must."`
 			label plan
-			`	He introduces you to the three Kimek, and goes to explain the plan. "Granted access to our civilian ships, you have been. Three new Kimek Spires, you have very recently bought. Many passengers to <destination>, in them you are carrying. A very lucrative job, it is." He pauses, and gives the Kimek permission to head out. "Pose as civilians, working in your ships, they will. Armed and ready to investigate <planet> thoroughly, the soldiers in those Spires are. Instructed to follow your ship there, these three were."`
+			`	He introduces you to the three Kimek, and goes to explain the plan. "Been granted access to our civilian ships, you have. Three new Kimek Spires, you have very recently bought. Carrying many passengers to <destination>, in them you are. A very lucrative job, it is." He pauses, and gives the Kimek permission to head out. "Pose as civilians, working in your ships, they will. Armed and ready to investigate <planet> thoroughly, the soldiers in those Spires are. Instructed to follow your ship there, these three were."`
```

Comment:
```suggestion
			`	He introduces you to the three Kimek and proceeds to explain the plan. "Been granted access to our civilian ships, you have. Three new Kimek Spires, you have very recently bought. Carrying many passengers to <destination>, in them you are. A very lucrative job, it is." He pauses and gives the Kimek permission to head out. "Pose as civilians, working in your ships, they will. Armed and ready to investigate <planet> thoroughly, the soldiers in those Spires are. Instructed to follow your ship there, these three were."`
```

Reply:
You can keep the comma if you reword it like this:
`He pauses for a moment, and he gives the Kimek permission to head out.`

You can't have a comma in this context unless there's an independent clause on the other end. Adding the subject `he` creates an independent clause, and I think the text flows better if you also add `for a moment` (though that last bit is not grammatically required).